ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -721,6 +721,16 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f4-disasterrecovery
```

Comment:
vphere does NOT have much resources to bear many concurrent jobs, you can see most of vsphere ci jobs are on f14, f28, so pls discuss with your team to lower its frequency.

Reply:
@wangke19 how do you think which frequency is make sense?

Reply:
Let's choice f14 for test.

Reply:
After we get a stable test results, we can reduce fequency .

Reply:
updated.